templates/advantage/_stripe-modal.html

```diff
@@ -76,7 +76,7 @@ <h2 class="p-modal__title js-contract-name u-no-margin--bottom" id="modal-title"
                   </label>
 
                   <label class="p-radio--inline">
-                    <input type="radio" class="p-radio__input" name="buying_for" id="buying_for_an_organisation" {% if account %}checked=""{% endif %} aria-labelledby="organisation">
+                    <input type="radio" class="p-radio__input" name="buying_for" id="buying_for_an_organisation" {% if account %}checked=""{% endif %} aria-labelledby="organisation" checked="true">
```

Comment:
The checked attribute is duplicated here I think we can remove the first instance `{% if account %}checked=""{% endif %}`

Reply:
@ClementChaumel good catch. Fixed!